src/meshapi/tests/test_lookups.py

```diff
@@ -0,0 +1,118 @@
+import json
+
+from django.contrib.auth.models import User
+from django.test import Client, TestCase
+
+from ..models import Member
+from .sample_data import sample_member
+
+
+class TestMemberLookups(TestCase):
```

Comment:
i love you

Reply:
Again, it was a bit selfish, I needed coverage for my new code 🙃